x-pack/plugins/security_solution/public/management/mocks/response_actions_http_mocks.ts

```diff
@@ -63,6 +67,29 @@ export const responseActionsHttpMocks = httpHandlerMockFactory<ResponseActionsHt
       return { data: response };
     },
   },
+  {
+    id: 'actionList',
+    path: ENDPOINTS_ACTION_LIST_ROUTE,
+    method: 'get',
+    handler: ({ path }): ActionListApiResponse => {
+      const response = new EndpointActionGenerator('seed').generateActionDetails();
+
+      // use the ID of the action in the response
+      response.id = path.substring(path.lastIndexOf('/') + 1) || response.id;
```

Comment:
I'm confused here... Why does the Action List API have an action id in the URL?

Maybe you forgot to remove this (when you copy+paste from the actionDetails mock defintion)?? 😀

Reply:
yeah looks like copy paste error